CONTRIBUTING.md

```diff
@@ -301,6 +301,39 @@ with an environment variable:
 JUJU_NOTEST_MONGOJS=1 go test github.com/juju/juju/...
 ```
 
+Conventional commits
+--------------------
+
+Once you have written some code and have tested the changes, the next step is to
+`git commit` it. For commit messages Juju follows the conventional commits guidelines specified [in this
+document](https://docs.google.com/document/d/1SYUo9G7qZ_jdoVXpUVamS5VCgHmtZ0QA-wZxKoMS-C0/).
```

Comment:
This document is not available outside of Canonical

Reply:
This one is. Its got the public tag at the top

Reply:
I think you need to reconfigure the sharing options
![image](https://github.com/juju/juju/assets/22304952/262ab537-475a-4b42-a96b-4cb62a736d97)